snap/gadget.go

```diff
@@ -47,7 +47,7 @@ type GadgetVolume struct {
 // type when we actually handle these.
 
 type VolumeStructure struct {
-	Label       string          `yaml:"label"`
+	Label       string          `yaml:"filesystem-label"`
```

Comment:
Do we need to preserve backwards compatibility here?

Reply:
We don't use this data currently, we just parse it.